tests/unit/APNSPayloadTest.php

```diff
@@ -119,8 +119,8 @@ public function testThatCustomDataSetterWorks() {
 			'baz' => [ 1.0, 1, '1', 0x1 ],
 		];
 		$push = $this->new_payload()->setCustomData( $custom_data );
-		$object = $this->encode_to_array( $push );
-		unset( $object['aps'] );
-		$this->assertEquals( $custom_data, $object );
+		$object = $this->to_stdclass( $push );
```

Comment:
`encode_to_array` was only used in one place, so I got rid of it and am using objects instead for consistency.